pkg/util/version.go

```diff
@@ -1,20 +1,57 @@
 package util
 
-import "regexp"
+import (
+	extVersion "github.com/hashicorp/go-version"
+	"regexp"
+	"strings"
+)
 
 //We limit to four digits... Git commits tend to be only nummeric as well
 var versionWithOptionalMinorAndPatch = regexp.MustCompile(`^[0-9]{1,3}(\.[0-9]+(\.[0-9]+)?)?$`)
```

Comment:
Her kan 3 økes til 5 på major for å være alignet med hva AOS gjør.

Reply:
Done